packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableSelector.ts

```diff
@@ -0,0 +1,72 @@
+import DragAndDropHelper from '../../../pluginUtils/DragAndDropHelper';
+import TableEditorFeature from './TableEditorFeature';
+import { createElement, normalizeRect } from 'roosterjs-editor-dom';
+import { KnownCreateElementDataIndex, SizeTransformer } from 'roosterjs-editor-types';
+
+const TABLE_SELECTOR_LENGTH = 12;
+const TABLE_SELECTOR_ID = '_Table_Selector';
+
+/**
+ * @internal
+ */
+export default function createTableSelector(
+    table: HTMLTableElement,
+    sizeTransformer: SizeTransformer,
```

Comment:
So this will be a little conflict with my PR https://github.com/microsoft/roosterjs/pull/755.

Whoever merge the last needs to fix this part to avoid use sizeTransformer.